Added Actions to Option Values

andrea-ale-sbarra opened the pull request.

Commando18 commented:
I have sent you a new PR: [#844](https://github.com/JorgeFerrer/liferay-portal/pull/844)

Commit:
LPS-121137 Add Ready for publication toggle

Commit:
LPS-121137 Add API to persist new setting

Commit:
LPS-121137 Do not propagate the changes when readyForPropagation is false and it’s not a reset from the Site or new Site creation

Commit:
LPS-121137 Apply changes in DefaultLayoutLayoutSetPrototypeLocalServiceWrapper

Commit:
LPS-121137 Apply in LayoutSetPrototypeStagedModelDataHandler

Commit:
COMMERCE-5142 Fix Brazilian currency label

Commit:
COMMERCE-5142 Aut SF

Commit:
COMMERCE-3885 add date range filter on create date in orders

Commit:
COMMERCE-3885 label for orders create date range

Commit:
COMMERCE-3885 buildLang

balazssk committed:
LPS-124190 Add weak reference to roleGroups

riccardo-alberti committed:
COMMERCE-3683 Show used price in cart jsps

riccardo-alberti committed:
COMMERCE-3683 Show used price in cart mini jsps

riccardo-alberti committed:
COMMERCE-3683 Fix key in cart total

riccardo-alberti committed:
COMMERCE-5239 if unit price is lower than promo then do not show promo price

riccardo-alberti committed:
COMMERCE-3683 Add promo price to order tables

cgoncas committed:
LPS-123084 Calculate the segments only when there are experiences associated with the layout.

pavel-savinov committed:
LPS-124407 Layout crawler to avoid creating dummy http request and response

pavel-savinov committed:
LPS-124407 Use new layout crawler to get page content for indexer

pavel-savinov committed:
LPS-124407 Remove method, we don't use it anymore

pavel-savinov committed:
LPS-124407 Log warning when layout content cannot be rendered in the crawler

jorgediaz-lr committed:
LPS-124310 portal-search-web: Enable nested queries in Custom Filter portlet

https://issues.liferay.com/browse/LPS-124310

jorgediaz-lr committed:
LPS-124310 portal-search: Enable nested queries in Custom Filter portlet

https://issues.liferay.com/browse/LPS-124310

victorg1991 committed:
LPS-124407 Always use the crawler

pavel-savinov committed:
LPS-124407 Set appropriate language cookie

jorgediaz-lr committed:
LPS-124310 portal-search: New test for the nested queries in ComplexQueryBuilderImpl

https://issues.liferay.com/browse/LPS-124310

marcosapmf committed:
LPS-124337 allow invalid available locales for property in journal

marcosapmf committed:
LPS-124337 allow invalid available locales for property in DL

marcosapmf committed:
LPS-124337 use default site locale if form default locale is null

marcosapmf committed:
LPS-124337 clean previous previous value when changing editing language id

marcosapmf committed:
LPS-124337 update only the changed field property in state

riccardo-alberti committed:
COMMERCE-3683 Call display context methods

riccardo-alberti committed:
COMMERCE-3683 Move business logic to display context

peerkar committed:
LPS-124583 portal-search-elasticsearch7-impl: Set score sort order

marcosapmf committed:
LPS-124337 test fix

riccardo-alberti committed:
COMMERCE-4835 Add sku to query pre process configuration

riccardo-alberti committed:
COMMERCE-4835 Sort keys alphabetically

ealonso committed:
LPS-124653 Removes constants since those are duplicated in DDMFormFieldType

ealonso committed:
LPS-124653 Uses DDMFormFieldType instead of DDMImpl

ealonso committed:
LPS-124653 Uses DDMFormFieldType instead of string

ealonso committed:
LPS-124653 build.gradle

ealonso committed:
LPS-124653 After the migration we should take into account also "image" as infoDisplayFieldType

Commit:
COMMERCE-5273 data set display component now works in commerce frontend static env

michaelhashimoto committed:
LRCI-1839 Add ability to get the TopLevelBuild from spira build results

michaelhashimoto committed:
LRCI-1839 Add method to update spira custom property values using an array

michaelhashimoto committed:
LRCI-1839 Update functional test case components, priority, & team name in spira results

michaelhashimoto committed:
LRCI-1839 Cache spira test case components for the spira result importers

alee8888 committed:
LPS-122000 Add base Local Service methods for Regions

alee8888 committed:
LPS-122000 Add Address reference entity

alee8888 committed:
LPS-122000 build-service-portal

slnn committed:
LPS-124576 Fix ResourcePermission for DDLRecordSetModel

slnn committed:
LPS-124576 Set compayId for DDMStructureLinkModel

slnn committed:
LPS-124576 DDL should not use the structureVersionId of DL

slnn committed:
LPS-124576 Correct index

marcosapmf committed:
LPS-124337 update settings context when changing field type

Commit:
LPS-121137 Autogenerated - ant build-service-portal

dacousalr committed:
LPS-123778 Override methods to observe the case of publishing subsequent scheduled Web Contents by getting their id_ as classPK instead of their resourcePrimKey

Commit:
LPS-121137 Actions - Ready for propagataion, Disable propagation

Commit:
LPS-121137 Language keys

Commit:
LPS-124353 Create formatDate util

Commit:
LPS-124353 Create activity section components

Commit:
LPS-124354 Apply activity section on edit and view entry pages

alee8888 committed:
LPS-122000 Add base Remote Service methods for Regions

alee8888 committed:
LPS-122000 Implement Remote Service methods

slnn committed:
LPS-124576 Also update ResourcePermission for DDMTemplateModel and DDMStructureModel according to current head of master

jbalsas committed:
LPS-122956 Removes portal-template-soy modules

This effectively removes server-side support for Soy rendering, meaning that
there won't be an available template manager for soy templates in the server.

This doesn't mean that compiled soy templates to JavaScript won't work which
reamin as a viable path forward if a developer decides to use them

jbalsas committed:
LPS-122956 Removes server-side soy rendering from SoyRenderer implementation

This basically means that the server will render an empty placeholder
and then components (if passed) will be hydrated client-side.

Additionally, we introduce a temporary workaround to reduce cumulative
layout shift. If a `__placeholder__` property is passed in the `context`
object, the renderer will use that instead of an empty placeholder so tag
implementors can choose to manually generate it temporarily.

Commit:
LPS-121137 Add help message to the active toggle

Commit:
LPS-121137 gw buildLang

matuzalemsteles committed:
LPS-124795 Fixes error when not listing nested fields for rules in Data Engine

drewbrokke committed:
LPS-121998 sql: copies add column statements to update-7.3.0-7.4.0-country.sql

drewbrokke committed:
LPS-121998 portal-impl: removes SQL statements that were copied

drewbrokke committed:
LPS-121998 sql: copies setting default boolean values to update-7.3.0-7.4.0-country.sql

drewbrokke committed:
LPS-121998 portal-impl: removes copied SQL

drewbrokke committed:
LPS-121998 portal-impl: removes extra DB call

jonathanmccann committed:
LPS-124808 Only escape when necessary. This makes it so the page's title does not appear as encoded.

alee8888 committed:
LPS-122000 Add tests

alee8888 committed:
LPS-122000 Implement Local Service methods

gabrielwas committed:
LPS-124591 Override form field required property with layout field required property

jbalsas committed:
LPS-122956 Provides a minimal Management Toolbar markup placeholder

This markup is used server-side to provide initial markup as a fallback now
that we're no longer rendering soy to limit layout cumulative shift when the
component hydrates client-side

This commit still leaves a slight shift because an empty toolbar has different
height than one with content. This is a small adjustment to adress later either
by agreeing on a fixed height or by rendering additional elements in the
placeholder

jbalsas committed:
LPS-122956 Documents breaking changes

jbalsas committed:
LPS-122966 Deprecates remaining Soy supporting classes

tinatian committed:
LPS-124751 Always process non-restrict template synchronously for better performance

tinatian committed:
LPS-124751 Force to recreate a new instance each time configuration is changed so that to initialize all the attributes.

tinatian committed:
LPS-124751 Fix unit test

drewbrokke committed:
LPS-124842 sql: quotes values since the number field of Country is a VARCHAR data type

istvansajtos committed:
LPS-124605 portal-kernel: Not all available locales are returned by BaseIndexer

https://issues.liferay.com/browse/LPS-124605

nkkwon committed:
LRQA-63476 Added path for open form and table view

nkkwon committed:
LRQA-63476 Added macros for open form and table view

nkkwon committed:
LRQA-63476 Added EditFormAndTableView to testcase

nkkwon committed:
LRQA-63476 Auto SF

wcao20170619 committed:
LPS-124583 portal-search-elasticsearch7-impl:copy Petteri's fix from
LPS-124581 for fixing Geodistance sort bug.
Script sort bug is also fixed in the commit.

https://issues.liferay.com/browse/LPS-124583

liuxingye committed:
LPS-122348 Add macro for add video from DM tab

liuxingye committed:
LPS-122348 Update macro for embed external video

liuxingye committed:
LPS-122348 Update macro for select depot asset

liuxingye committed:
LPS-122348 Add tests for embed external video

liuxingye committed:
LPS-122348 Add tests for embed video with staging

dgarciasarai committed:
LPS-124870 Change Click Goal section and use a ClayInput to display the selected target

dgarciasarai committed:
LPS-124870 Build Lang

dgarciasarai committed:
LPS-124870 Manage state.selectedTarget without #. Display it manually if applies. Database goalTarget keeps the same format

marcoscv-work committed:
LPS-102099 - Fix compatibility product menu styles

marcoscv-work committed:
LPS-102099 - Move toolbar property

marcoscv-work committed:
LPS-102099 - Move form-builder-page-header style

dgarciasarai committed:
LPS-124870 Fix tests

Commit:
COMMERCE-5332 data set actions dropdown can now highlight items

marcoscv-work committed:
LPS-102099 - Move ddm form page style

Commit:
COMMERCE-5329 double fetch removed when data set display gets invoked

marcoscv-work committed:
LPS-102099 - Remove compat-liferay

antonio-ortega committed:
LPS-124654 Adding a mask on top of video (only in edit mode) so in case the user click on the fragment it prevents to play the video and behaves the same than any other fragment

marcoscv-work committed:
LPS-102099 - Fix simulation sidebar styles

jeyvison committed:
LPS-124591 SF

adolfopa committed:
LPS-123324 Check resource for translated model and language

Translation entries don't have resources themselves, they depend on the
translated model resources and/or language resource.

hhuijser committed:
LPS-105380 Add missing @Override

jbalsas committed:
LPS-122956 Uses id instead of componentId and autogenerates it if it doesn't exist

This is necessary because in some contexts the toolbar lacks an ide
at this point. This can happen if no id or componentId is set and the
toolbar is not linked to a search container

sandrodw3 committed:
LPS-124926 Not rely on Keep Open property when fragment is within master layout

wcao20170619 committed:
LPS-124583 portal-search-elasticsearch7-impl: add unit test for ElasticsearchSortFieldTranslator.

https://issues.liferay.com/browse/LPS-124583

shuyangzhou committed:
LPS-124751 Expose getter from base class.

dacousalr committed:
LPS-124941 Adjust path to the new value set in LPS-115709

michaelhashimoto committed:
LRCI-1839 Add method to check if a build is completed

michaelhashimoto committed:
LRCI-1839 Add caching for for finding axis builds & test results

michaelhashimoto committed:
LRCI-1839 Add helper methods within BaseSpiraArtifact in order to be used within results

michaelhashimoto committed:
LRCI-1839 Add method to get key id directly from a spira artifact

michaelhashimoto committed:
LRCI-1839 Make getCustomPropertyJSONObject a public method to be used in results

michaelhashimoto committed:
LRCI-1839 Make assigning spira test sets a public method

michaelhashimoto committed:
LRCI-1839 Add methods to get spira test sets from spira build results

michaelhashimoto committed:
LRCI-1839 Add methods to get portal release & pull request objects from build results

michaelhashimoto committed:
LRCI-1839 Clean up logic to instantiate a spira test case run

tinatian committed:
LPS-124751 Add properties for async support, prepare for next

tinatian committed:
LPS-124751 buildLang

tinatian committed:
LPS-124751 Add support to process freemarker template asynchronously, timeout <= 0 means disable, timeout > 0 means enable

tinatian committed:
LPS-124751 Enable async on ci

shuyangzhou committed:
LPS-124751 Move to static util

rafaprax committed:
LPS-121676 Use the query filter context instead

michaelhashimoto committed:
LRCI-1839 Add caching for getting test classes from axis builds

gabrielwas committed:
LPS-124701 Save native object data layout as non-default

michaelhashimoto committed:
LRCI-1839 Add methods to get more information from a spira result object

michaelhashimoto committed:
LRCI-1839 Add spira test result for top level builds

michaelhashimoto committed:
LRCI-1839 Cache spira automation hosts in the spira importer

michaelhashimoto committed:
LRCI-1839 Fix null pointer errors that occur when setting up the spira importer

brianikim committed:
LPS-124521 Make the items more readable in mobile view.

BryanEngler committed:
LPS-124310 portal-search: Also assert that the nested query's query is a BooleanQuery

https://issues.liferay.com/browse/LPS-124310

pei-jung committed:
LPS-123926 Show -- for the accounts count for Guest account group

pei-jung committed:
LPS-123926 Hide action menu for Guest account group

pei-jung committed:
LPS-123926 Disable checkbox for Guest account group

pei-jung committed:
LPS-123926 Remove link on Guest account group

Commit:
LPS-124948 Modify locator for card sticker

michaelhashimoto committed:
LRCI-1839 Add spira test result values class hierarchy

Commit:
LPS-124948 Run case on acceptance

michaelhashimoto committed:
LRCI-1839 Add methods to get batch custom property values

Commit:
LPS-124948 Add macro to view card sticker

Commit:
LPS-124948 Remove abandoned macro

Commit:
LPS-124948 Use new macro to assert mark as default

Commit:
LPS-124948 Modify case to view react fragment has a visual marker in vertical card

liuxingye committed:
LPS-124893 Add path for preview video body in WC

liuxingye committed:
LPS-124893 Update macro for embed video in WC

liuxingye committed:
LPS-124893 Add test for embed video in WC

brianchandotcom committed:
LPS-124584 auto SF

brianchandotcom committed:
LPS-121998 match service.xml

michaelhashimoto committed:
LRCI-1839 Add spira test result details class hierarchy

brianchandotcom committed:
LPS-124190 as used

michaelhashimoto committed:
LRCI-1839 Add methods to get spira test result details & values

brianchandotcom committed:
LPS-121676 as used in 'new UpdateByQueryDocumentRequest'

michaelhashimoto committed:
LRCI-1839 Add methods to create spira test runs on spira test results

Commit:
LRQA-63760 Modify cases to add web content via JSONWS

brianchandotcom committed:
LPS-124751 Wordsmith

brianchandotcom committed:
LPS-124751 Sort

brianchandotcom committed:
LPS-124751 Makes more sense like this

victorg1991 committed:
LPS-124407 Rename

Commit:
LRQA-63814 Remove duplicated steps

Commit:
LRQA-63814 Modify color value

Commit:
LRQA-63814 Replace locator to assert value of color field

victorg1991 committed:
LPS-123778 Add override annotation

Commit:
LRQA-63814 Modify macro to assert options checked

4lejandrito committed:
LPS-124868 Revert "LPS-124868 sort?"

This order was on purpose. Originally we were taking the v= value as
everything to the right including parameters which was wrong. With those
tests we were ensuring this wouldn't happen again.

This reverts commit d0cab889e43f897d611392b3d69c53456bc8385b.

cgoncas committed:
LPS-123084 SF, per Brian's comment in https://github.com/brianchandotcom/liferay-portal/pull/96947#issuecomment-745587801.

Commit:
LRQA-63814 Modify parameter

cgoncas committed:
LPS-123084 Better name.

hhuijser committed:
LPS-124961 Remove suppression for GradleDependenciesCheck

hhuijser committed:
LPS-124961 Apply GradleDependenciesCheck

jbalsas committed:
LPS-122956 Waits for toolbar to be ready before attempting to focus on the search input

Commit:
LPS-122603 Move questions configuration to message boards section

hhuijser committed:
LPS-124961 Remove suppression for GradleTestDependencyVersionCheck

hhuijser committed:
LPS-124961 Apply GradleTestDependencyVersionCheck

jbalsas committed:
LPS-122956 Provides a different class selector for test path

dgarciasarai committed:
LPS-124870 Apply feedback

dgarciasarai committed:
LPS-124870 Rename tests

Commit:
LRQA-63814 Modify locator for Select from List field

Commit:
LRQA-63814 Modify macro to assert selected value in Select from List field

Commit:
LRQA-63813 Modify case to assert success message

hhuijser committed:
LPS-124961 Remove suppressions for JavaInternalPackageCheck

Jeremy-Ch committed:
LPS-123421 Add macro for viewing the video entry in item selector

hhuijser committed:
LPS-124961 Move *ConfigurationBeanDeclaration to *.internal.settings.definition

hhuijser committed:
LPS-124961 Move *ConfigurationPidMapping to *.internal.configuration.definition

Jeremy-Ch committed:
LPS-123421 Add test for the video shortcut

hhuijser committed:
LPS-124961 Update find-security-bugs-false-positives.txt

Commit:
LRQA-63816 Remove abandoned part

hhuijser committed:
LPS-124961 Remove suppression for JSONPackageJSONDependencyVersionCheck

Commit:
LRQA-63817 Modify parameters

hhuijser committed:
LPS-124961 Remove suppression for ListUtilCheck

hhuijser committed:
LPS-124961 Pass arguments directly to 'ListUtil.fromArray'

hhuijser committed:
LPS-124961 Simplify by using 'ListUtil.fromArray'

ealonso committed:
LPS-124653 Add missing import

carolmoreschi committed:
LRQA-63548 Remove unnecessary .releng entries

adolfopa committed:
LPS-124949 Ignore null results received from Sharepoint

jkappler committed:
LPS-124723 We have to throw an exception if there are any failures

adolfopa committed:
LPS-124964 Remove mismatching scriptlet marker

hhuijser committed:
LPS-124961 Sort

Commit:
COMMERCE-5335 data set display side panel fixed on rtl

sandrodw3 committed:
LPS-124910 Hide menu when there is no elements

jkappler committed:
LPS-124730 Correct markup

Commit:
COMMERCE-5142 Auto SF

naraandrade committed:
LRQA-61739 Remove test from quarantine

boton committed:
LPS-124950 Hide padding and border to win space in DM sidebar

rafaprax committed:
LPS-124889 Create an intermediate path(1.4.1 -> 1.4.2, 1.4.2 -> 2.0.0) using the DummyUpgradeProcess to cover the upgrade from 7.1.x

rafaprax committed:
LPS-124889 Check if the change has been applied previously

liferay-translation committed:
LPS-77699 Update Translations

cgoncas committed:
LPS-119476 Support inputs of type submit to be targeteable as click goals.

pei-jung committed:
LPS-123925 Hide the roles tab for Guest account

liferay-translation committed:
LPS-77699 Update Translations

pei-jung committed:
LPS-122000 Throw exception for non-existing region

petershin committed:
LPS-124985 Fix classpath for Gradle 6.6.1

mtambara committed:
LPS-124887 More robust path handling to account for windows

pei-jung committed:
LPS-122000 Update regionId on organizations when region is removed

pei-jung committed:
LPS-122000 This should be countryId

pei-jung committed:
LPS-122000 SF - validate after getting the object

pei-jung committed:
LPS-122000 Simplify tests

BryanEngler committed:
LPS-124605 portal-search: Also apply in DefaultSelectedFieldNamesQueryConfigContributor

https://issues.liferay.com/browse/LPS-124605

Commit:
LRQA-63476 - Small improvements

shuyangzhou committed:
LPS-124992 Add finder

shuyangzhou committed:
LPS-124992 Regenerate

brianchandotcom committed:
Revert "COMMERCE-5142 Auto SF"

This reverts commit 2c745af46253941a3529ee765f8d13d6b597248f.

brianchandotcom committed:
Revert "COMMERCE-5142 Aut SF"

This reverts commit f83531ca8827794e69c74cf080f79a8a3b9e041a.

shuyangzhou committed:
LPS-124992 Add per company limits support for group(site), organization, role, team and user.

shuyangzhou committed:
LPS-124887 SF

liuxingye committed:
LRQA-63831 Update assertion

brianchandotcom committed:
LPS-124310 Sort

brianchandotcom committed:
LPS-124992 Sort

brianchandotcom committed:
LPS-124992 Wordsmith

brianchandotcom committed:
LPS-124992 Wordsmith

brianchandotcom committed:
LPS-124992 SF

brianchandotcom committed:
LRCI-1839 prep next

brianchandotcom committed:
LPS-124653 Sort

brianchandotcom committed:
LPS-121137 wordsmith

brianchandotcom committed:
LPS-121137 regen

brianchandotcom committed:
LPS-124576 sort

brianchandotcom committed:
LPS-124576 as used

brianchandotcom committed:
Baseline

andrea-ale-sbarra committed:
COMMERCE-5364 headless-commerce-admin-catalog-impl added action to OptionValue (yaml file)

andrea-ale-sbarra committed:
COMMERCE-5364 headless-commerce-admin-catalog-impl added action to OptionValue

andrea-ale-sbarra committed:
COMMERCE-5364 headless-commerce-admin-catalog-impl autogenerated code (gw buildREST)
